playbooks/roles/kubernetes/README.md

```diff
@@ -0,0 +1,120 @@
+|k8s_pvc_labels		    |               |       [“billingType”, “region”, “zone”]       | 
+|k8s_service	        | loadbalancer	|                                               |
+|k8s_service_annot		|               |       [“ip-type”,”zone”,”vlan”]               |
+|k8s_list_domain_ip	    | localhost 	|                                               |
+|k8s_networking		    |               |       ingress                                 |
+|k8s_gateway_domain		|               |       *.nio.io                                |
+|k8s_discovery_domain	|               |       *.nio.io                                |
+
+
+### More details about the environmental variables: 
+
+**ansible_user**: z/OS host user name
+
+**ansible_password**: z/OS host password
+
+**ansible_ssh_host**: z/OS host IP address / domain
+
+**kubeconfig**: Path to an existing Kubernetes config file. If not provided, and no other connection options are provided, it load the default configuration file from host's ~/.kube/config.json.
+
+**k8s_context**: The name of a context found in the config file. 
+
+**convert_for_k8s**: Location of the script file to create ConfigMaps and Secrets used by Zowe containers. This script is found in the z/OS system. i.e. convert_for_k8s=<zowe-instance-dir>/bin/utils/convert-for-k8s.sh
+
+**k8s_storageclass**: Zowe's PVC has a default StorageClass value (=hostpath) that may not apply to all Kubernetes clusters. Check and provide the storageClassName. You can use kubectl get sc to confirm which StorageClass you can use.
+
+**k8s_pv_name**: The name of the persistence volume name. If you provide PV name, it will be the main place to look for volume information (surpressing storageclass).
+
+**k8s_pvc_labels**: IBM Cloud requires additional information about pvc through labels. You can get the information from https://cloud.ibm.com/docs/containers?topic=containers-file_storage#file_qs . Please provide in array format: k8s_pvc_labels: [“hourly”, “us-south”, “dal12”]
+- billingType . For example, hourly.
+- Region. Found in here: (https://cloud.ibm.com/kubernetes/clusters/"cluster-id"/overview?region=us-south&resourceGroup="resource-group-id")
+- Zone. You should be able to see this zone in your cluster home information page (using above url)
+
+**k8s_service**: Type of Service. Default is “loadbalancer”. Please use below table to check which service you’ll need based on the Kubernetes provider.
+
+|Kubernetes provider |	k8s_service             |
+|:-------------------|:-------------------------|
+|docker-desktop	     | LoadBalancer             |
+|bare-metal	         | LoadBalancer or NodePort |
+|cloud-vendors	     | LoadBalancer             |
+|OpenShift	         | LoadBalancer or NodePort |
+
+**k8s_service_annot**: Used specifically for IBM Cloud Kubernetes, where addition information about the load-balancer provided using array. For example, please provide value in array format - k8s_service_annot: [“public”, “dal12”, “12345”]. 
```

Comment:
More generically, not sure if we can use `object` instead of `array`.

If it's `object`, the user can put this in the host var file:

```
k8s_service_annot:
  service.kubernetes.io/ibm-load-balancer-cloud-provider-ip-type: public
  service.kubernetes.io/ibm-load-balancer-cloud-provider-zone: dal12
  service.kubernetes.io/ibm-load-balancer-cloud-provider-vlan: "12345"
```